### DISCUSSION TITLE
Require status checks to pass before merging is not disabled

### DISCUSSION THREAD

**Discussion by Edberg-Vasan** [community/community#30777] [Actions]
Hi guys,
I protect the **private** branch by configuring Github actions to require the status check to pass before the pull request is mergeable.

First I have a pull request which runs CI checks but it gots failed. In my case, CI is not needed. So I disabled the "**Require status checks to pass before merging**" but CI checks are not disabled.

Whenever I create a pull request against the private branch, CI checks are running. I don't know why. Please give some suggestions.

**Comment by airtower-luna** [comment]
The "Require status checks to pass before merging" setting doesn't have anything to do with whether any checks *run*, it only blocks merging if the configured checks failed or didn't run at all.

If you don't want the checks to run, you need to adjust the configuration of those checks. For example, if you're using Actions set a branch filter on `push` or `pull_request` events.
